### PR TITLE
admin & admin2: Added /clearchat command

### PR DESCRIPTION
**Pull request by AfuSensi**
#168 
Added /clearchat to both admin and admin2.
Default access rights for Moderator, SuperModerator and Admin.

### CODE REVIEW

**Comment by Dezash** [comment]
Every other command in the admin resource has a GUI equivalent, perhaps there could be a button in the server tab for this feature as well?

File: [admin]/admin2/conf/ACL.xml [hunk 1]
```diff
@@ -69,13 +69,14 @@
     <right name="command.setskygradient" access="true" />
     <right name="command.setgamespeed" access="true" />
     <right name="command.setgravity" access="true" />
-	<right name="command.setfpslimit" access="true" />
+	  <right name="command.setfpslimit" access="true" />
```
Comment (qaisjp) [inline [admin]/admin2/conf/ACL.xml]:
```suggestion
    <right name="command.setfpslimit" access="true" />
```

File: [admin]/admin2/server/admin_functions.lua [hunk 2]
```diff
@@ -469,6 +469,10 @@ aFunctions = {
         end,
         ["shutdown"] = function(reason)
             shutdown(iif(reason, tostring(reason), nil))
+        end,
+        ["clearchat"] = function()
+            clearChatBox()
+            return true
```
Comment (ecastro98) [inline [admin]/admin2/server/admin_functions.lua]:
```suggestion
```

File: [admin]/admin2/server/admin_functions.lua [hunk 3]
```diff
@@ -469,6 +469,10 @@ aFunctions = {
         end,
         ["shutdown"] = function(reason)
             shutdown(iif(reason, tostring(reason), nil))
+        end,
+        ["clearchat"] = function()
+            clearChatBox()
```
Comment (ecastro98) [inline [admin]/admin2/server/admin_functions.lua]:
```suggestion
            return clearChatBox()
```

Comment (AfuSensi) [reply [admin]/admin2/server/admin_functions.lua]: clearChatBox() returns false for getRootElement().

**Comment by AfuSensi** [comment]
I messed up and commited to my fork, causing it to show up in this PR. I reverted it so i hope it wont matter.

File: [admin]/admin2/conf/ACL.xml [hunk 4]
```diff
@@ -150,13 +151,14 @@
...
```
Comment (qaisjp) [inline [admin]/admin2/conf/ACL.xml]:
```suggestion
     <right name="command.setfpslimit" access="true" />
```

File: [admin]/admin2/conf/ACL.xml [hunk 5]
```diff
@@ -309,13 +312,14 @@
     <right name="command.setskygradient" access="false" />
     <right name="command.setgamespeed" access="false" />
     <right name="command.setgravity" access="false" />
-	<right name="command.setfpslimit" access="false" />
+	  <right name="command.setfpslimit" access="false" />
```
Comment (qaisjp) [inline [admin]/admin2/conf/ACL.xml]:
```suggestion
     <right name="command.setfpslimit" access="false" />
```

File: [admin]/admin2/conf/ACL.xml [hunk 6]
```diff
@@ -231,13 +233,14 @@
     <right name="command.setskygradient" access="true" />
     <right name="command.setgamespeed" access="true" />
     <right name="command.setgravity" access="true" />
-	<right name="command.setfpslimit" access="true" />
+	  <right name="command.setfpslimit" access="true" />
```
Comment (qaisjp) [inline [admin]/admin2/conf/ACL.xml]:
```suggestion
   <right name="command.setfpslimit" access="true" />
```

Comment (AfuSensi) [reply [admin]/admin2/conf/ACL.xml]: I just realised this is a tab vs spaces issue, my bad.
I was wondering why it looked properly indented in my editor, but not on github.

**Comment by qaisjp** [comment]
looks good, do you still intend to add the button? if not, i can merge as it is right now

**Comment by AfuSensi** [comment]
Yes i do. If these button placements are ok i'll add it.

**admin**
admin1button

**admin2**
admin2button

**Comment by qaisjp** [comment]
yeah looks good ❤️

**Comment by AfuSensi** [comment]
Great, i've added it to this pull request.

**Pull request merged**